pkg/pullsync/pullsync.go

```diff
@@ -179,7 +181,7 @@ func (s *Syncer) SyncInterval(ctx context.Context, peer swarm.Address, bin uint8
 			s.logger.Debug("storage has", "error", err)
 			continue
 		}
-		if !have {
+		if !have && bin >= s.radius.StorageRadius() {
```

Comment:
so what we want is the proximity order of the address >= storage radius (the address at line 171)

Reply:
Done